manual-tests.md

```diff
@@ -123,7 +123,7 @@
 
 1. Download https://developer.qualcomm.com/download/db410c/linux-board-support-package-v1.2.zip
 2. Extract it and copy the file `firmware.tar` to the directory `demos/96boards-kernel`.
-3. Run `snapcraft --target-arch arm64` in the `demos/96boards-kernel` directory.
+3. Run `snapcraft snap --target-arch arm64` in the `demos/96boards-kernel` directory.
```

Comment:
This is necessary because the behavior of options changed with the new CLI. Perhaps this should wait until we've discussed this properly?